angles.py

```diff
@@ -1 +1,19 @@
-print ("angle calculator")
\ No newline at end of file
+import math
+#import numpy as np
+import matplotlib.pyplot as mpl
+
+def sin_calc(start, end, step):
+    y_values = []
+    x_values = []
+    for x in range(start * 10, end * 10, step):
```

Comment:
Instead of multipying by 10, you should use a numpy range function that will allow arbitrary floats. other wise it will break if you use a factor other than 1/10